src/main/java/com/salesforce/dynamodbv2/mt/mappers/MtAmazonDynamoDbLogger.java

```diff
@@ -64,7 +64,7 @@ private MtAmazonDynamoDbLogger(MtAmazonDynamoDbContextProvider mtContext,
                                    Consumer<List<String>> logCallback,
                                    List<String> methodsToLog,
                                    boolean logAll) {
-        super(mtContext, amazonDynamoDb, null);
+        super(mtContext, amazonDynamoDb, null, null, null);
```

Comment:
I figured forcing instances to null these values consciously is better than overloading a constructor default nulling this. Open to doing that overload given the defaults works for 99.9% of instances I bet